### PR TITLE
Arxivce 1503 update arxiv docs readme

### PR DESCRIPTION
**Pull request by alisonhofer**
Added more screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,56 +28,70 @@ familiar with this ask a dev or an admin to help you.
...
 
 - Create a "new branch for this request and start a pull request" will be selected
-- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`
+- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`.
 
-![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)
+<kbd>![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)</kbd>
 
-- **Please note:** The above step will only put your changes on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
+- **Please note:** The above step will only put your changes on a branch on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
```
Comment (kyokukou) [inline README.md:57]: a branch on develop isnt really correct. develop is a branch, your changes will be on a different branch

File: README.md [hunk 2]
```diff
@@ -28,56 +28,70 @@ familiar with this ask a dev or an admin to help you.
...
 
 - Your pull request will appear at the top of the page in a yellow box. Click on the blue button "Compare & pull request".
 
-![Screenshot how to compare changes in the pull request in github](/non-info/06-help-comparePR.png)
+<kbd>![Screenshot how to compare changes in the pull request in github](/non-info/06-help-comparePR.png)</kbd>
 
 - Add a title for your pull request and a description if the title is not self explanatory.
-- To the right, click on the gear icon to the right of "Reviewers"
+- To the right, click on the gear icon to the right of "Reviewers".
 - A dropdown will appear and you can request people to review your PR.
```
Comment (kyokukou) [inline README.md:72]: any commentary on who should be requested to review it? (I don't know what the answer would be here)

File: README.md [hunk 3]
```diff
@@ -28,56 +28,70 @@ familiar with this ask a dev or an admin to help you.
...
+<kbd>![Screenshot of how to select reviewers of your PR and create the PR in Github](/non-info/09-help-mergePR.png)</kbd>
+
+- You will be prompted to confirm you decision, click "Confirm merge".
+
+  <kbd>![Screenshot of how to select reviewers of your PR and create the PR in Github](/non-info/10-help-confirmMerge.png)</kbd>
 
-### 3rd: Merge the PR to `master` to Deploy Changes
 
+### 4th: Deploy your changes to `master`
 To get your changes deployed, merge it to the branch called `master`. This will
```
Comment (kyokukou) [inline README.md:95]: I'm pretty sure this should also be a PR that needs to be approved

**Review by kyokukou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alisonhofer) [reply README.md:59]: Got it. I moved this comment after one merges their branch into develop. It makes more sense there.

File: README.md (hunk 2 above)
Comment (alisonhofer) [reply README.md]: I'll add a note.

**Pull request closed without merging**